pkg/operators/ustack/ustack.go

```diff
@@ -53,12 +56,25 @@ func (o *Operator) GlobalParams() api.Params {
 	return nil
 }
 
+func (o *Operator) ParamDescs() params.ParamDescs {
+	return params.ParamDescs{
+		{
+			Key:          goReSymParam,
+			Description:  "Symbol tables for stripped Go executables in GoReSym json format",
+			DefaultValue: "",
+		},
+	}
+}
+
```

Comment:
Implement this directly in InstanceParams(). The `ParamDescs` method is legacy and shouldn't be needed by new operators.